lib/libc/minimal/include/sys/_types.h

```diff
@@ -14,5 +14,6 @@
 
 typedef int64_t _TIME_T_;
 typedef int32_t _SUSECONDS_T_;
+typedef uint32_t _CLOCK_T_;
```

Comment:
Thanks for using the same type on 32- and 64- bit systems. However, I wonder if (given the 1000000 value for CLOCKS_PER_SEC) whether this would be way more useful as a 64-bit type? 4295 seconds isn't very long.

Reply:
I can change that to 64-bit

Reply:
Not sure why this is still 32-bit here I'll fix that too